packages/core/contracts/optimistic-asserter/implementation/escalation-manager/BaseEscalationManager.sol

```diff
@@ -1,16 +1,36 @@
 pragma solidity 0.8.16;
 
+import "@openzeppelin/contracts/access/Ownable.sol";
 import "../../interfaces/EscalationManagerInterface.sol";
+import "../../interfaces/OptimisticAsserterInterface.sol";
 
 /**
  * @title BaseEscalationManager
  * @notice Base contract for escalation managers. This contract is responsible for managing the escalation policy for
  * assertions. This base implementation simply exposes the required interface and provides a default implementation
  * (returning default values or doing nothing).
  */
-contract BaseEscalationManager is EscalationManagerInterface {
+contract BaseEscalationManager is EscalationManagerInterface, Ownable {
```

Comment:
Why does this contract now need to be ownable? It doesn't look like it's used here.

Reply:
I think that @md0x pulled this up the inheritance path to this base contract. You'll see he remove this from the imports and inheritance path on the other contracts.

Reply:
Good question. Initially I implemented the `optimisticAsserter` as a settable variable in the BaseEscalationManager, in that case I needed the contract to be ownable. Finally I made the optimisticAsserter immutable and set during construction. 

I realised that pulling up from Ownable to BaseEscalationManager could make sense as all the other EMs use it but maybe it's odd if BaseEscalationManager doesn't. Wdyt?

After thinking about it, I think it is better to go back to the way it was before and stick to the changes strictly necessary to correct the OZ issue.

Reply:
removed Ownable from BaseEscalationManager